core/src/main/java/com/opensymphony/xwork2/util/ProxyUtil.java

```diff
@@ -18,13 +18,20 @@
  */
 package com.opensymphony.xwork2.util;
 
+import com.opensymphony.xwork2.ognl.DefaultOgnlCacheFactory;
+import com.opensymphony.xwork2.ognl.OgnlCache;
+import com.opensymphony.xwork2.ognl.OgnlCacheFactory;
 import org.apache.commons.lang3.reflect.ConstructorUtils;
 import org.apache.commons.lang3.reflect.FieldUtils;
 import org.apache.commons.lang3.reflect.MethodUtils;
+import org.hibernate.proxy.HibernateProxy;
```

Comment:
This isn't acceptable solution as it means everybody will have to use Hibernate just to satisfy this dependency.

Reply:
The dependency is optional and the check silently skipped :)

Reply:
Sorry, missed the "skip" :/